login.php

```diff
@@ -0,0 +1,20 @@
+<?php
+session_start ();
```

Comment:
`session_start()` no space between letters and brackets

Reply:
Good spot.